examples/create_checks_from_csv/check.py

```diff
@@ -12,7 +12,7 @@
 
 ###########################################################
 # TODO: Provide your API Key, keep this secure
-lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+lob.api_key = ''
```

Comment:
dunno if you want this to be an empty string or something like `'REPLACE_ME'`

Reply:
`YOUR_API_KEY` is good for all